attack/scripts/bashrc

```diff
@@ -61,6 +61,44 @@ info() {
 } 1>&2
 readonly -f info
 
+#
+# Starting point.
+#
+
+starting_point() {
+  # Identify the starting point mode.
+  MODE="$(sed '3,3!d' /tasks.yaml)"
+  export MODE
+
+  #Determine the type.
+  if [[ $MODE =~ "internal-instance" ]]; then
+    ssh_internal
+  else
```

Comment:
If you `return` from the `if` block so that you don't have to use else and indent further which makes the code more readable.  See https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88 for a fuller rationale :)

Reply:
Covered in the new commit.